context/authContext.tsx

```diff
@@ -1,36 +1,45 @@
-import React, { useEffect } from 'react';
-import AsyncStorage from '@react-native-async-storage/async-storage';
+import React, { createContext, useEffect, useState } from 'react';
+import * as SecureStore from 'expo-secure-store';
 
```

Comment:
was doing a little reading, and i think AsyncStorage is technically secure, it's just not encrypted and i guess is vulnerable to jailbroken phones. TIL
https://stackoverflow.com/questions/39028755/save-sensitive-data-in-react-native

Reply:
I didn't realize you commented!

I kind of went down a rabbit hole with that too. Some people are very opinionated about storing tokens in `AsyncStorage`.